codeofconduct.html

```diff
@@ -0,0 +1,98 @@
+ <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+ <link rel="stylesheet" href="./assets/css/global.min.css" />
+ <link rel="stylesheet" href="./assets/css/header.min.css" />
+ <link rel="stylesheet" href="./assets/css/footer.min.css" />
+ <link rel="stylesheet" href="./assets/css/about.min.css" />
+ <script src="https://kit.fontawesome.com/a5c8bf6536.js" crossorigin="anonymous"></script>
+ <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,400i,600,600i,700,700i&display=swap"
+   rel="stylesheet" />
+ <script src="./assets/js/app.js"></script>
+
+ <!-- Global site tag (gtag.js) - Google Analytics -->
+ <script async src="https://www.googletagmanager.com/gtag/js?id=G-26B6FB8T8P"></script>
+ <script>
+   window.dataLayer = window.dataLayer || [];
+   function gtag(){dataLayer.push(arguments);}
+   gtag('js', new Date());
+   gtag('config', 'G-26B6FB8T8P');
+ </script>
+</head>
+
+<body>
+            <div class="container">
+
+                <h1>Code of Conduct</h1>
+              
+                <p>This document is adapted from Code for America’s Code of Conduct and Anti-Harrassment Policy.</p>
+                
+                <hr>
+              
+                <h3>Code for San Jose's Code of Conduct</h3>
+              
+                <p>The Code for San Jose community expects that Code for San Jose network activities, events, and digital forums:</p>
+              
+                <ol>
+                  <li>Are a safe and respectful environment for all participants.</li>
+                  <li>Are a place where people are free to fully express their identities.</li>
+                  <li>Presume the value of others. Everyone’s ideas, skills, and contributions have value.</li>
+                  <li>Don’t assume everyone has the same context, and encourage questions.</li>
+                  <li>Find a way for people to be productive with their skills (technical and not) and energy. Use language such as “yes/and”, not “no/but.”</li>
+                  <li>Encourage members and participants to listen as much as they speak.</li>
+                  <li>Strive to build tools that are open and free technology for public use. Activities that aim to foster public use, not private gain, are prioritized.</li>
+                  <li>Prioritize access for and input from those who are traditionally excluded from the civic process.</li>
+                  <li>Work to ensure that the community is well-represented in the planning, design, and implementation of civic tech. This includes encouraging participation from women, minorities, and traditionally marginalized groups.</li>
+                  <li>Actively involve community groups and those with subject matter expertise in the decision-making process.</li>
+                  <li>Ensure that the relationships and conversations between community members, the local government staff and community partners remain respectful, participatory, and productive.</li>
```

Comment:
Missing a comma here before the `and`. Should be:

`Ensure that the relationships and conversations between community members, the local government staff, and community partners remain respectful, participatory, and productive.`